tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_migrated_valid_invalid.py

```diff
@@ -0,0 +1,288 @@
+"""
+EOF validation tests for EIP-3540 migrated from
+ethereum/tests/src/EOFTestsFiller/EIP3540/validInvalidFiller.yml
+"""
+
+import pytest
+
+from ethereum_test_tools import EOFTestFiller
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools.eof.v1 import Container, EOFException, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+
+from .. import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-3540.md"
+REFERENCE_SPEC_VERSION = "8dcb0a8c1c0102c87224308028632cc986a61183"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+@pytest.mark.parametrize(
+    "eof_code,exception",
+    [
+        pytest.param(
+            # Deployed code without data section
+            Container(
+                name="EOF1V3540_0001",
+                sections=[
+                    Section.Code(code=Op.PUSH1[0] + Op.POP + Op.STOP, max_stack_height=1),
+                ],
+            ),
+            None,
```

Comment:
I am getting exceptions here and exception names mismatch. 
Does this PR require most recent evmone build? 
if so to measure the coverage I must update the evmone docker image